src/stories/roundCheckbox.js

```diff
@@ -0,0 +1,7 @@
+import "./roundCheckbox.css";
```

Comment:
Need to change link to styles from your component folder (06_shared/ui/input/checkbox)

Reply:
it doesn't work when i'm just import them from component folder - storybook component became blank, when i delete css file from storybook

Reply:
Need to init package https://storybook.js.org/recipes/sass